pages/common/gource.md

```diff
@@ -0,0 +1,28 @@
+# gource
+
+> Renders an animated tree diagram of Git, SVN, Mercurial and Bazaar repositories.
+> It shows files and folders being created, modified or removed over time.
+
+- Run gource in a directory (assuming the command is run from the project's root directory):
+
+`gource {{path/to/repository}}`
+
+- Set the resolution:
```

Comment:
So `(assuming the command is run from the project's root directory)` should be moved to this description?